data/mods/EW_Pack/ew_mods.json

```diff
@@ -1,23 +1,44 @@
 [
     {
+        "id": "autofire_654",
+        "type": "GUNMOD",
+        "name": "rifle auto-fire mechanism",
+        "description": "A mechanism that converts your semi-automatic 5.45x39mm or 7.62x39mm rifle to selective fire.  It is a modular unit.",
+        "weigth": 250,
+        "volume": 1,
+        "integral_volume": 0,
+        "price": 90000,
+        "material": [ "steel" ],
+        "symbol": ":",
+        "color": "red",
+        "location": "mechanism",
+        "mod_targets": [ "rifle" ],
+        "acceptable_ammo": [ "545x39", "762" ],
+        "install_time": 20000,
+        "loudness_modifier": 1,
+        "mode_modifier": [ [ "AUTO", "auto", 10 ] ],
+        "min_skills": [ [ "weapon", 3 ], [ "mechanics", 2 ] ]
+    },
+    {
         "type" : "GUNMOD",
-        "id" : "autofire_654",
-        "name" : "handmade auto-fire mechanism",
+        "id" : "imp_autofire_654",
```

Comment:
The gunmod was split into a player-craftable version, which cannot be removed, and a modular version. The original id is used for the modular version.